tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+	"compilerOptions": {
+		"module": "umd",
+		"noImplicitAny": true,
+		"removeComments": true,
+		"preserveConstEnums": true,
+		"sourceMap": true,
+		"target": "es5"
```

Comment:
I'll turn this on, I was actually disappointed when it didn't warn me about checking nullable but didn't think to look for an option to enable it